### ISSUE TITLE
[asciidoc] html alts: make html output friendlier for accessibility and text-based browsers

### ISSUE THREAD

**Issue by 0-issue** [riscv/riscv-isa-manual#1098]
For users who use accessibility features like text-to-voice, as well as text-based browsers the html output of ISA document can be improved in multiple ways:

1. Use better alt for images. For latex equation, this could be as simple as copying latex equation in alt if you cannot come up with a  better alt text.
2. Use unicode glyphs for simple cases instead of svgs where they are not really needed. For instance use unicode [horizontal ellipses](https://www.compart.com/en/unicode/U+2026) instead of using stem+latex formula ``latexmath:[$\cdot\cdot\cdot$]``, as latter is a heavy hammer; similarly use unicode [X mark](https://en.wikipedia.org/wiki/X_mark#Unicode) instead of ``latexmath:[$\times$]``, etc.
3. The images should be available as assets in a folder, seems like they are being generated on-the-fly in browser? This isn't supported by simpler or text-based browsers.

Example:

In section 1.5.1, a simple "x" sign could have been one of the unicode [X mark](https://en.wikipedia.org/wiki/X_mark#Unicode) instead of being an svg image, with an intelligible alt string instead of ``alt="stem 715efe442a3a591fd63a05098adb4ab7"``.

Screenshot:
<img width="91" alt="Screenshot 2023-08-20 at 8 11 07 AM" src="https://github.com/riscv/riscv-isa-manual/assets/25681815/098dd4f4-9a9f-4f1a-9e31-3ecbed50c616">

Generated html code:
```
5 <span class="image"><img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0i
    aHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3Ln
    czLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iMjguMDc3NDg5cHQiIGhlaWdodD0iNi43MzI0
    MjJwdCIgdmlld0JveD0iMCAwIDI4LjA3NzQ4OSA2LjczMjQyMiIgdmVyc2lvbj0iMS4xIj
    4KPGRlZnM+CjxnPgo8c3ltYm9sIG92ZXJmbG93PSJ2aXNpYmxlIiBpZD0iZ2x5cGgwLTAi
    Pgo8cGF0aCBzdHlsZT0ic3Ryb2tlOm5vbmU7IiBkPSJNIDAuNTkzNzUgMi4xMjUgTCAwLj
    U5Mzc1IC04LjQ2ODc1IEwgNi41OTM3NSAtOC40Njg3NSBMIDYuNTkzNzUgMi4xMjUgWiBN
    IDEuMjY1NjI1IDEuNDUzMTI1IEwgNS45Mzc1IDEuNDUzMTI1IEwgNS45Mzc1IC03Ljc4MT
    I1IEwgMS4yNjU2MjUgLTcuNzgxMjUgWiBNIDEuMjY1NjI1IDEuNDUzMTI1ICIvPgo8L3N5
    bWJvbD4KPHN5bWJvbCBvdmVyZmxvdz0idmlzaWJsZSIgaWQ9ImdseXBoMC0xIj4KPHBhdG
    ggc3R5bGU9InN0cm9rZTpub25lOyIgZD0iTSA4LjQwNjI1IC02LjQ2ODc1IEwgNS42ODc1
    IC0zLjc1IEwgOC40MDYyNSAtMS4wNjI1IEwgNy43MzQzNzUgLTAuNDA2MjUgTCA1LjAzMT
    I1IC0zLjA5Mzc1IEwgMi4zMjgxMjUgLTAuNDA2MjUgTCAxLjY1NjI1IC0xLjA2MjUgTCA0
    LjM1OTM3NSAtMy43NSBMIDEuNjU2MjUgLTYuNDY4NzUgTCAyLjMyODEyNSAtNy4xMjUgTC
    A1LjAzMTI1IC00LjQzNzUgTCA3LjczNDM3NSAtNy4xMjUgWiBNIDguNDA2MjUgLTYuNDY4
    NzUgIi8+Cjwvc3ltYm9sPgo8L2c+CjwvZGVmcz4KPGcgaWQ9InN1cmZhY2U2MzYiPgo8Zy
    BzdHlsZT0iZmlsbDpyZ2IoMCUsMCUsMCUpO2ZpbGwtb3BhY2l0eToxOyI+CiAgPHVzZSB4
    bGluazpocmVmPSIjZ2x5cGgwLTEiIHg9IjkuMDExNzE5IiB5PSI3LjEzMDg1OSIvPgo8L2
    c+CjwvZz4KPC9zdmc+Cg==" alt="stem 715efe442a3a591fd63a05098adb4ab7" wi
    dth="28" height="6"></span> 16-bit
```

An accessibility user will hear something like ``5 stem 715efe442a3a591fd63a05098adb4ab7 16-bit`` instead of ``5 x 16-bit``, similar will be the plight of a user who reads this in a text-based web browser.
